https://invent.kde.org/pim/itinerary/-/commit/1beaf403181bfa4414c506a588ecda1425403451.patch

From 1beaf403181bfa4414c506a588ecda1425403451 Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Fri, 26 Jul 2024 16:47:10 +0200
Subject: [PATCH] Explicitly search for and link against QtKeychain

This is used by the Traewelling integration directly now, but so far was
was only used indirectly via libQuotient.
---
 .kde-ci.yml            | 1 +
 CMakeLists.txt         | 1 +
 src/app/CMakeLists.txt | 1 +
 3 files changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f70187eb9..f7b32e6e0 100644
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -57,6 +57,7 @@ find_package(KPim6Itinerary 5.23.0 CONFIG REQUIRED)
 find_package(KPublicTransport 23.07.70 CONFIG REQUIRED)
 find_package(KOSMIndoorMap CONFIG REQUIRED)
 find_package(KHealthCertificate CONFIG)
+find_package(Qt6Keychain CONFIG REQUIRED)
 find_package(LibIcal 3.0)
 find_package(ZLIB REQUIRED)
 set(QUOTIENT_FORCE_NAMESPACED_INCLUDES ON)
diff --git a/src/app/CMakeLists.txt b/src/app/CMakeLists.txt
index 4b7947714..d8c5bdcf7 100644
--- src/app/CMakeLists.txt.orig
+++ src/app/CMakeLists.txt
@@ -250,6 +250,7 @@ target_link_libraries(itinerary-app PRIVATE
     KF6::UnitConversion
     Qt::QuickControls2
     KCalendarCoreExtras
+    Qt6Keychain::Qt6Keychain
 )
 if (ANDROID)
     target_link_libraries(itinerary PUBLIC KAndroidExtras PRIVATE Qt::CorePrivate)
-- 
GitLab


